hrjobcontract/CRM/Hrjobcontract/Import/Form/DataSource.php

```diff
@@ -36,24 +36,74 @@
 /**
  * This class gets the name of the file to upload
  */
-class CRM_Hrjobcontract_Import_Form_DataSource extends CRM_Hrjobcontract_Import_Form_DataSourceBaseClass {
-  public $_parser = 'CRM_Hrjobcontract_Import_Parser_Api';
+class CRM_Hrjobcontract_Import_Form_DataSource extends CRM_Import_Form_DataSource {
+  const PATH = 'civicrm/job/import';
+  const IMPORT_ENTITY = 'Import Contracts';
+
   protected $_enableContactOptions = FALSE;
-  protected $_userContext = 'civicrm/job/import';
-  protected $_mappingType = 'Import Job';
-  protected $_entity = array('HRJobContract', 'HRJobContractRevision', 'HRJobDetails', 'HRJobPay', 'HRJobHealth', 'HRJobPension', 'HRJobHour', 'HRJobLeave');
+
   /**
-  * Include duplicate options
-  */
+   * Include duplicate options
+   */
   protected $isDuplicateOptions = FALSE;
 
-    /**
+
+  protected $_entity = array('HRJobContract', 'HRJobContractRevision', 'HRJobDetails', 'HRJobPay', 'HRJobHealth', 'HRJobPension', 'HRJobHour', 'HRJobLeave');
```

Comment:
Where are $_enableContactOptions, $isDuplicateOptions and $_entity being used? I checked both CRM_Import_Form_DataSource and CRM_Core_Form (from civicrm-core  4.7.7 branch) and it seems they are not inherited from them. I also checked if this class is extended by another and didn't find any. So, since it seems they are declared here and not used anywhere else, maybe we don't need them? Am I missing something?